### PR TITLE
PMP::Add function triangle(face_descriptor, tmesh)

### PR DESCRIPTION
**Pull request by afabri**
## Summary of Changes

Add a convenience function to obtain a triangle for a face.

## Release Management

* Affected package(s): PMP

### CODE REVIEW

**Comment by MaelRL** [comment]
Rule  which has already been violated through regularize_face_selection_borders, copy_face_graph, generators, etc.

Maybe BGL should embrace the fact that it serves as a base for our BGL-ized algorithms rather than having an awkward depency to PMP-Core say in AABB_tree (if e.g. the property map Triangle_from_face_descriptor were to use the function introduced by this PR).

File: Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangle.h [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  /*!
+   * \ingroup PkgPolygonMeshProcessingRef
+   *
+   * computes a triangle for a face descripor of a triangle mesh.
+   *
+   * @tparam TriangleMesh a model of `FaceGraph`
+   * @tparam NamedParameters a sequence of \ref bgl_namedparameters "Named Parameters"
+   *
+   * @param tmesh the triangulated surface mesh
+   * @param np an optional sequence of \ref bgl_namedparameters "Named Parameters" among the ones listed below
```
Comment (albert-github) [inline Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangle.h:37]: Looks like you forgot to document `fd`, see also overnight builds:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-5.5-Ic-92/include/CGAL/Polygon_mesh_processing/triangle.h:33: warning: The following parameter of CGAL::Polygon_mesh_processing::triangle(typename boost::graph_traits< TriangleMesh >::face_descriptor fd, const TriangleMesh &tmesh, const NamedParameters &np=parameters::default_values()) is not documented:
  parameter 'fd'
```

**Comment by sloriot** [comment]
Successfully tested in CGAL-5.5-Ic-95

**Comment by afabri** [comment]
@MaelRL did we say that we expect the triangle vertices to be in the same order?

**Comment by sloriot** [comment]
What do you mean by same order? the normal of the triangle is different from that of the face?

**Comment by lrineau** [comment]
What is the status of this PR? It is https://github.com/CGAL/cgal/labels/Tested but https://github.com/CGAL/cgal/labels/Not%20yet%20approved (added by @sloriot). What is not approved?

**Comment by sloriot** [comment]
We can merge it. I was wondering if this shouldn't go in BGL but we should do a global pass on things from PMP that should go in BGL.

**Pull request merged**